platform/util/platform-util/src/main/java/org/codice/ddf/platform/util/TransformerProperties.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.platform.util;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.Set;
+
+import javax.xml.transform.ErrorListener;
+import javax.xml.transform.OutputKeys;
+
+/**
+ * Properties for transforming XML
+ */
+public class TransformerProperties {
+    private Map<String, String> transformProperties;
+
+    private ErrorListener errorListener;
+
+    public TransformerProperties() {
+        transformProperties = new HashMap<>();
+    }
+
+    public void addOutputProperty(String propertyType, String propertyValue) {
+        transformProperties.put(propertyType, propertyValue);
+    }
+
+    public void prettyFormat() {
```

Comment:
This seems very odd to me to have a method that mutates the internal map.  I was imagining there would be a prettyPrint method in `XMLUtils`.  If you are against a pretty print method in XMLUtils then I could also see this method becoming a static factory method that returns a new TransformerProperties preloaded with these two properties.